packages/web-components/src/customized-built-ins/image/image.styles.ts

```diff
@@ -0,0 +1,62 @@
+import { css } from '@microsoft/fast-element';
+import {
+  borderRadiusCircular,
+  borderRadiusMedium,
+  colorNeutralStroke2,
+  shadow4,
+  strokeWidthThin,
+} from '../../theme/design-tokens.js';
+
+/** Image styles
+ *
+ * @public
+ */
+export const styles = css`
+  img[is='fluent-image'] {
+    contain: content;
+    box-sizing: border-box;
+    min-height: 8px;
```

Comment:
Is there a reason why we want this min height?

Reply:
Let me revisit the history here - all I did here was pull and augment the existing image styles. Could be unintentional :)